jooby/src/main/java/org/jooby/handlers/AssetHandler.java

```diff
@@ -185,11 +187,20 @@ public AssetHandler cdn(final String cdn) {
   }
 
   /**
-   * @param maxAge Set the cache header max-age value in seconds.
+   * @param maxAge Set the cache header max-age value.
    * @return This handler.
    */
-  public AssetHandler maxAge(final long maxAge) {
-    this.maxAge = maxAge;
+  public AssetHandler maxAge(final Optional<Duration> maxAgeOpt) {
```

Comment:
don't think we need this method: ```Optional<Duration>```. If we allow a user to set this... why allow an possible empty object? We shouldn't.

Also and to keep backward compatibility.. we should keep the ```long``` version as it was.